CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 
 - [cronos#502](https://github.com/crypto-org-chain/cronos/pull/502) Fix failed tx are ignored in json-rpc apis.
 - [cronos#556](https://github.com/crypto-org-chain/cronos/pull/556) Bump gravity bridge module version to include bugfixes (including grpc endpoint)
+- [cronos#611](https://github.com/crypto-org-chain/cronos/pull/611) Fix mistake on acknowledgement error in ibc middleware.
```

Comment:
Should be put in `State Machine Breaking` section?

Reply:
yes its consensus breaking, changed